packages/blaze/attrs.js

```diff
@@ -273,10 +273,16 @@ if (Meteor.isClient) {
   anchorForNormalization = document.createElement('A');
 }
 
+const _protocolCache = new Map(); // Simple cache
```

Comment:
Please add before this a comment with a summary that explains how the caching is expected to work, because in 1+ year we need to understand what the next 100 lines of code do.